modules/cnf-use-device-interrupt-processing-for-isolated-cpus.adoc

```diff
@@ -6,4 +6,16 @@
 [id="use-device-interrupt-processing-for-isolated-cpus_{context}"]
 = Upgrading the performance profile to use device interrupt processing for isolated CPUs
 
-When you upgrade the Performance Addon Operator performance profile custom resource definition (CRD) from v1 to to v2, `GloballyDisableIrqLoadBalancing` is set to *true* on existing profiles.
+When you upgrade the Performance Addon Operator performance profile custom resource definition (CRD) from v1 to to v2, `GloballyDisableIrqLoadBalancing` is set to `"true"` on existing profiles.
```

Comment:
`true`

Reply:
The annotation requires quotes in this context though - the yaml is incorrect without the quotes. I was unsure about this one tbh.

Reply:
Sounds good! If the quotes are required, then we can leave it as it is.